tests/global-fill-none-before.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="170" height="89" viewBox="0 0 170 89" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Thank you for your PR.

Please consider adding the test `fill=""` and `fill="transparent"`

Other LGTM.

Reply:
> test fill="" and fill="transparent"

Reply:
thanks

> test fill="" and fill="transparent"

but those are not valid SVG fill values. Also, it's not the content of `fill` attribute that i'm testing here, but the fact that it gets propagated down. I think this is sufficient.